styleguide.md

```diff
@@ -210,3 +210,13 @@ When a grammar rule is mentioned in the text, the rule shall be formatted with t
 ```
 The node shall contain a \lstinline[language=grammar]!stored-definition! that…
 ```
+
+## English natural language text
+
+This section gives guidelines for how the natural language text in English should be written.
+
+The text is written in American English.
```

Comment:
I'm not sure if this is fully correct, and I believe we often had a more complicated description. 
Clearly we often use American **spelling** (meter, color &c).

Reply:
OK, but I think it makes sense to state as much as we can.  If it's clear that we use American spelling, what would the reason be to not commit to American English more generally?

Reply:
I believe that was discussed previously with a more complicated conclusion, but couldn't find that.
For MSL we recently found no conclusion https://github.com/modelica/ModelicaStandardLibrary/issues/2750

Reply:
OK, that's a very relevant background.  Maybe one needs to be more pragmatic and take the view that also American English isn't a static thing; they used to spell it _behavior_, but nowadays they tend to spell it _behaviour_ instead?

Reply:
Wouldn't it be simpler to just write that 'The text primarily uses American English spelling' and not overstate this?

Reply:
I agree that such a formulation would be better than trying to define our own dialect of Popular Modeling English.

However, having read more on this topic in various places, I've become convinced that the right thing to do is to simply pick American English, even though it doesn't come with the most popular spelling for every word.  It seems to generally be considered the only way of being perceived as consistent, which I find much more important than being able to pick _behaviour_ just because it's an increasingly popular spelling.

Reply:
There are a number of other minor differences; but it seems simpler to just close this and hope that it will not lead to too many changes.